rootfs/rootfs/usr/local/etc/init.d/docker

```diff
@@ -5,78 +5,38 @@
 #import settings from profile (e.g. HTTP_PROXY, HTTPS_PROXY)
 test -f '/var/lib/boot2docker/profile' && . '/var/lib/boot2docker/profile'
 
-: ${DOCKER_HOST:='-H tcp://0.0.0.0:2375'}
-: ${DOCKER_TLS:=auto}
 : ${DOCKER_STORAGE:=auto}
 : ${DOCKER_DIR:=/var/lib/docker}
 : ${DOCKER_ULIMITS:=1048576}
 : ${DOCKER_LOGFILE:=/var/lib/boot2docker/docker.log}
 
-: ${CERTDIR:=/var/lib/boot2docker/tls/}
-: ${CERT_INTERFACES:='eth0 eth1'}
+: ${DOCKER_TLS:=auto} # 'no' or 'auto'
 : ${CACERT:="${CERTDIR}ca.pem"}
-: ${CAKEY:="${CERTDIR}cakey.pem"}
 : ${SERVERCERT:="${CERTDIR}server.pem"}
 : ${SERVERKEY:="${CERTDIR}serverkey.pem"}
```

Comment:
`docker-machine` provisions the key as `/var/lib/boot2docker/server-key.pem`.  The filename here doesn't match.